packages/admin/cms-admin/src/dam/config/DamConfigContext.tsx

```diff
@@ -1,11 +1,14 @@
 import * as React from "react";
 
+import { DamRow } from "../DataGrid/FolderDataGrid";
+
 export interface DamConfig {
     additionalMimeTypes?: string[];
     scopeParts?: string[];
     enableLicenseFeature?: boolean;
     requireLicense?: boolean;
     additionalToolbarItems?: React.ReactNode;
+    renderAdditionalRowContent?: (row: DamRow) => React.ReactNode;
```

Comment:
I decided to add the `render` prefix because this is a render prop. However, it's kinda odd that `additionalToolbarItems` isn't a render prop. What do you think?

Reply:
yes, they could both use the same technique

Reply:
what is the difference between the first two?